web/files/referencedby.go

```diff
@@ -10,6 +11,53 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+// ContentType is the official mime-type for JSON-API
+const ContentType = "application/vnd.api+json"
```

Comment:
I would prefer to avoid re-declaring this content-type. You can use `jsonapi.ContentType` (from `pkg/jsonapi`) instead.

Reply:
I hesitated to do that in the first place. Will do.